build.sh

```diff
@@ -36,10 +38,11 @@ do
 
   echo 'Processing '$GST_LIB
   cd $GST_LIB
-  sed -i 's?libdir=.*?libdir='`pwd`'?g' pkgconfig/*
-  sed -i 's?.* -L${.*?Libs: -L${libdir} -lgstreamer_android?g' pkgconfig/*
-  sed -i 's?Libs:.*?Libs: -L${libdir} -lgstreamer_android?g' pkgconfig/*
-  sed -i 's?Libs.private.*?Libs.private: -lgstreamer_android?g' pkgconfig/*
+  sed -i -e 's?libdir=.*?libdir='`pwd`'?g' pkgconfig/*
+  sed -i -e 's?.* -L${.*?Libs: -L${libdir} -lgstreamer_android?g' pkgconfig/*
+  sed -i -e 's?Libs:.*?Libs: -L${libdir} -lgstreamer_android?g' pkgconfig/*
+  sed -i -e 's?Libs.private.*?Libs.private: -lgstreamer_android?g' pkgconfig/*
+  rm -rf pkgconfig/*pc-e*
```

Comment:
This is reverting #5 cause I found that the script won't work properly on MacOS otherwise.